src/dbus/Makefile.am

```diff
@@ -28,7 +28,9 @@
 
 include $(abs_top_nlbuild_autotools_dir)/automake/pre.am
 
+if OTBR_ENABLE_DBUS_SERVER
```

Comment:
I suspect you'll encounter some build errors in `make check`. We can just have this library always built and it won't be linked into the final output if `OTBR_ENABLE_DBUS_SERVER` is. not enabled.

Reply:
OpenWRT doesn't have d-bus library.  And this library depends on dbus.